### ISSUE TITLE
Custom icons not functioning

### ISSUE THREAD

**Issue by SirFancyBacon** [hellbringer616/Strix_RPG#13]
Custom icons are not functioning in the skill tree; they appear to not be in a Minecraft friendly format; appears to be an issue with NT-RPG but not entirely sure yet.

**Comment by NeumimTo** [comment]
Right now theres only partial, not fully implemented support for this.

**Comment by SirFancyBacon** [comment]
@NeumimTo Currently i cannot get the arrows or "connecting lines" to show up with the texture pack; is that normal?

Also i have some (minor)texture pack experience, if the issue is the formatting of the Texturepack and not with your code i would love to manage this feature for you (i'm a bad artist; but i can get it working :))

**Comment by NeumimTo** [comment]
ah..these icons. 
Its in Settings.conf - 
SKILLTREE_RELATIONS
|,minecraft:stick,|,0
item name,itemid,character representation in the ascii map,damage value

SKILLTREE_BUTTON_CONTROLLS
 "North,minecraft:diamond_hoe,Up,1"
direction one of NWES,item id,item name,damage value

The resourcepack retextures damaged & unbreakable diamond hoes with damage value somewhere between 3-10

If the damage value is >1 the item will have nbt unbreakable tag.

**Comment by NeumimTo** [comment]
Also i have literally none texture pack know-how. Thats why i have no idea how to fix the missing texture for diamond_hoe item type. If you know how i will be happy if you create pull request to assets repository

**Comment by SirFancyBacon** [comment]
@NeumimTo 
I submitted a pull request for the Assets repo; i think it may fix the issue.
ResourcePack type was set to 1 instead of 3; so defacto it won't work with MC1.12.x

I also renamed some textures; If your plugin functions how i think it does; that means that it determines textures based on the name of the object; so "uparrow.png" won't work since the arrow is simply called "up" so i renamed the arrows to see if they work.

I've left the lines named the same to see if they will will function in the new format; if one works and the other does not; i change to that naming scheme :)


I'm working until 7PM -5GMT tonight so i won't beable to test it soon; should you have time to give it a look over and comment on the pull with any issues; i'll be happy to fix it!
